### PR TITLE
Runs the JavaDoc Generator on Source that has been Delombokified

### PR DESCRIPTION
**Pull request by JLLeitschuh**
This fixes the build failures occuring in this other PR: #1881

### CODE REVIEW

File: rewrite-java/build.gradle.kts [hunk 1]
```diff
@@ -48,16 +48,7 @@ dependencies {
...
     (options as StandardJavadocDocletOptions).addStringOption("Xdoclint:none", "-quiet")
-
-    // ChangePackage and OrderImports due to lombok error which looks similar to this:
-    //     openrewrite/rewrite/rewrite-java/src/main/java/org/openrewrite/java/OrderImports.java:42: error: cannot find symbol
-    // @AllArgsConstructor(onConstructor_=@JsonCreator)
-    //                     ^
-    //   symbol:   method onConstructor_()
-    //   location: @interface AllArgsConstructor
-    // 1 error
-    exclude("**/JavaParser**", "**/ChangePackage**", "**/OrderImports**")
```
Comment (JLLeitschuh) [inline rewrite-java/build.gradle.kts:62]: @aegershman ^

File: build-src/src/main/kotlin/org/openrewrite/gradle/Delombok.kt [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+            "delombok"(
+                "to" to outputDirectory.asFile.get().absolutePath,
+                "classpath" to compileClasspath.asPath
+            ) {
+                sourceFiles.addToAntBuilder(this, "fileset", FileCollection.AntType.FileSet)
+            }
+        }
+    }
+}
```
Comment (JLLeitschuh) [inline build-src/src/main/kotlin/org/openrewrite/gradle/Delombok.kt:71]: The number of magical Gradle incantations required to make tasks in a way that works with up-to-date checking is too high 😆

**Comment by JLLeitschuh** [comment]
@sambsnyd can you take a look at this when you get a moment? We need someone familiar with Gradle to look at it

File: build-src/src/main/kotlin/org/openrewrite/gradle/Delombok.kt (hunk 2 above)
Comment (sambsnyd) [reply build-src/src/main/kotlin/org/openrewrite/gradle/Delombok.kt:73]: True that

File: rewrite-java/build.gradle.kts (hunk 1 above)
Comment (sambsnyd) [reply rewrite-java/build.gradle.kts:62]: He doesn't work at Moderne anymore. Of course, he's welcome to make contributions to the project, but I wouldn't expect him to reply

**Pull request closed without merging**